search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -37,17 +35,50 @@ function onClickCountry(e) {
 function setCountryTrends(countryCode) {
   const topTrendsTab = document.getElementById('top-trends-tab');
   topTrendsTab.innerHTML = '';
+  fetch('/country-trends/' + countryCode).then(countryTrends =>
+    countryTrends.json()).then(trends => {
+      if (trends.length === 0) {
+        topTrendsTab.innerText = 'Trends in this country is not available.';
+      } else {
+        trends.forEach(trend => {
+          const trendHeader = document.createElement('h5');
+          trendHeader.innerText = trend.topic;
+          topTrendsTab.appendChild(trendHeader);
+        });
```

Comment:
```suggestion
  fetch('/country-trends/' + countryCode).then(countryTrends => 
      countryTrends.json()).then(trends => {
    if (trends.length === 0) {
      topTrendsTab.innerText = 'Trends are not available for the selected country.';
    } else {
      trends.forEach(trend => {
        const trendHeader = document.createElement('h5');
        trendHeader.innerText = trend.topic;
        topTrendsTab.appendChild(trendHeader);
      });
```

Reply:
use the version in master

Reply:
@carmenBen @ChenyuHeidiZhang I personally like Carmen's old comment better, but you guys can resolve this. I think the new formatting is correct, but Carmen feel free to change it.

Reply:
So based on this https://google.github.io/styleguide/jsguide.html#formatting-function-expressions link, it seems that line 39 should have a 4 space indent, and then lines 40 through 47 should be indented 2 spaces off from that. Does that seem right to you guys? I will change the comment now, but leave the indenting until we have a consensus.

Reply:
I didn't see the old comment, but sure we can do "line 39 should have a 4 space indent, and then lines 40 through 47 should be indented 2 spaces off from that." But change the innerText to 'Trends are not available for the selected country.'

Reply:
agreed, changed it